website/content/docs/commands/web.mdx

```diff
@@ -0,0 +1,84 @@
+---
+layout: docs
+page_title: Web REPL - Command
+description: |-
+  The Vault web user interface (UI) includes an advanced mode that mimics some basic commands from the Vault CLI. It can be useful a user is more familiar with the CLI or API as it lets a user directly input the paths they wish to manipulate.
+
+---
+
+
+# Web REPL
+
+The Vault web user interface (UI) includes an advanced mode that mimics some basic commands from the Vault CLI. It can be useful a user is more familiar with the CLI or API as it lets a user directly input the paths they wish to manipulate.
+
+~> **Note:** The Vault Web REPL is _not_ a full terminal emulator. Features like environment varialbes, HEREDOC, or piping of data or files will not work unless explicitly documented.
+
+## Command history
+The Web REPL will keep a history of all of the commands you've entered in your current session. If you refresh the browser or use the `clearall` command, this history will be reset.
+
+To cycle through the command history, you can use the up and down arrows when the REPL input has focus. If you reach either end of the history, it will cycle back to the the other side and continue looping through them.
+
+## Commands
+The Vault Web REPL implements the Create/Read/Update/Delete/List (CRUDL) commands from the Vault CLI. With these basic commands, a user can interact with most of the Vault API even if there aren't explicit screens for it in Vault's Web UI.
+
+All commands can optionally be prefixed with `vault` as if you were using the CLI - the intent being that a large number of example commands from the documentation should work via simply copy and pasting them from the documentation site into the web REPL.
+
+### delete
+`delete` in the Web REPL is the same as using the [`delete`](/vault/docs/commands/delete) command in the CLI. It can be used to delete secrets and configuration from Vault at the given path via an HTTP DELETE.
+
+### kv-get
+The Web REPL is a convenience method much like [`kv read`](/vault/docs/commands/kv/read) to read KV secrets from a version 2 KV secrets engine. The command `kv-get secret/foo` is functionally equivalent to running the REPL `read` command with the full API path: `read secret/data/foo`.
+
+There is also a `-metadata` flag that is shorthand for `-field=metadata`.
+
+### list
+The Web REPL `list` command is functionally identical to the CLI [`list`](/vault/docs/commands/list) command. It is used to list keys at a given path such as roles in an auth method, or keys for a given secrets engine. The Web REPL version issues an GET with the `?list=true` query parameter as described in the [API overview](/vault/api-docs#api-operations) since web browsers do not support custom HTTP verbs.
```

Comment:
```suggestion
The Web REPL `list` command is functionally identical to the CLI [`list`](/vault/docs/commands/list) command. It is used to list keys at a given path such as roles in an auth method, or keys for a given secrets engine. The Web REPL version issues a GET with the `?list=true` query parameter as described in the [API overview](/vault/api-docs#api-operations) since web browsers do not support custom HTTP verbs.
```